<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SampleGroupingDlg</class>
 <widget class="QDialog" name="SampleGroupingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="sortGroup">
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>检测类型：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>分组情况:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QListWidget" name="typeView"/>
      </item>
      <item row="1" column="1">
       <widget class="MyTableView" name="groupView"/>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="saveBtn">
          <property name="text">
           <string>保存分组</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="printBtn">
          <property name="text">
           <string>打印标签</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="printGroup">
     <property name="title">
      <string>打印确认</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QComboBox" name="typeSelectBox">
          <item>
           <property name="text">
            <string>全部类型</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>选择类型</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="siteSelectBox">
          <item>
           <property name="text">
            <string>全部点位</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>选择点位</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTextBrowser" name="typeBrowser"/>
        </item>
        <item row="1" column="1">
         <widget class="QTextBrowser" name="siteBrowser"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>选择周期：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="periodBox">
          <item>
           <property name="text">
            <string>全部周期</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>采样日期：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="printOkbtn">
          <property name="text">
           <string>确认</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelBtn">
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTableView</class>
   <extends>QTableView</extends>
   <header>mytableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
